Expensify/App
# Add VBA loading animation

cc @MitchExpensify 

### Details

Adds an animation in between VBA steps to make the UI more intuitive.


### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/172797

### Tests

1. Create a new workspace.
2. Navigate to it and click `Get Started`
3. Choose `Connect Manually`
4. Fill in the Routing number and Account number and click `Save & Continue`
5. Fill in all the company information and click `Save & Continue`
6. Make sure the loading page/animation looks like on the videos

Additionally, can you please test on physical Android device, if the gif animation really works between the steps of VBA flow? In emulator, it has been static image, but the entire app is very slow in my emulator so not sure what has caused it.

### QA Steps

Same as tests.

### Tested On

- [x] Android - only emulator

### Screenshots


#### Web


https://user-images.githubusercontent.com/36083550/128869377-da473484-c2a8-4031-9bae-a12e6f028acf.mov


#### Mobile Web


https://user-images.githubusercontent.com/36083550/128868297-3edf5cf7-020d-4d5a-bba5-1fff6419f26f.mov

#### Desktop


https://user-images.githubusercontent.com/36083550/128868268-b6d25d45-4216-40a8-b34e-694f8fd8c259.mov

#### iOS



https://user-images.githubusercontent.com/36083550/128868320-3f6e4b8d-cb39-4cd0-9e3d-39b5b2fc3e8b.mov


#### Android


https://user-images.githubusercontent.com/36083550/128873902-36663723-0121-4402-a6ab-14e3905710c7.mov
